src/main/webapp/dev/js/common/instructor.js

```diff
@@ -323,6 +326,26 @@ function bindSessionDeleteLinks() {
     });
 }
 
+function attachEventToSendInviteLink() {
```

Comment:
This works fine in the Course page, but doesn't work that well in the Students page. The confirmation modal is not showing up, and the entire page refreshes itself(check how it looks like in the Course remind page).

Reply:
It is working for me here, but I am moving towards attending to the changes you've requested above.

Reply:
Yup I checked again, it's working fine!